src/main/java/com/microsoft/azure/datalake/store/Core.java

```diff
@@ -635,7 +640,7 @@ public static void concat(String path,
             qp = new QueryParams();
             qp.add("deleteSourceDirectory", "true");
         }
-
+        opts.contentType = "application/json";
```

Comment:
opts is an input parameter. It is an error for customer to specify any value other than application/json in the incoming parameter value. It is best not to put this in RequestOptions structure (which is a set of input parameters), since asking for this as input is awkward. See comment beloe for how we can send json without having to make it an input parameter. Feel free to call if you need more details after the comment below,